src/tf/parse_batchnorm.cpp

```diff
@@ -38,16 +39,44 @@ struct parse_batchnorm : op_parser<parse_batchnorm>
     instruction_ref parse(const op_desc& /*opd*/,
                           const tf_parser& /*parser*/,
                           tf_parser::node_info info,
-                          const std::vector<instruction_ref>& args) const
+                          std::vector<instruction_ref> args) const
     {
-        float epsilon  = 1e-5f;
-        float momentum = 0.9f;
+        // different default epsilon than from ONNX
+        float epsilon = 1e-4f;
```

Comment:
Why is the tolerance lower on MIGraphX vs Onnx? Was that a specific design call made or were we hitting some error without this?

Reply:
This is just a different between the TensorFlow spec and the ONNX spec.

Reply:
Oh okay, so there's no way around it. Got it

Reply:
Are you using, this https://www.tensorflow.org/api_docs/python/tf/keras/layers/BatchNormalization 

or this : https://www.tensorflow.org/api_docs/python/tf/nn/batch_normalization

I assume, keras one. It uses epsilon = 0.001 == 1e-3f.